Hard code the version because it would require Git instead.

Upstream: not yet, need some discuss to find a proper alternative
--- a/CMakeLists.txt	2020-06-05 14:53:32.438653467 +0200
+++ b/CMakeLists.txt	2020-06-05 14:53:42.862042935 +0200
@@ -27,7 +27,7 @@
 endif ()
 
 set (PACKAGE_NAME "QjackCtl")
-set (PACKAGE_VERSION "${VERSION}")
+set (PACKAGE_VERSION "0.6.2")
 set (PACKAGE_BUGREPORT "rncbc@rncbc.org")
 set (PACKAGE_STRING "${PACKAGE_NAME} ${PACKAGE_VERSION}")
 set (PACKAGE_TARNAME "qjackctl")
